mypyc/primitives/registry.py

```diff
@@ -35,13 +35,21 @@
 optimized implementations of all ops.
 """
 
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, NamedTuple
 
 from mypyc.ir.ops import (
     OpDescription, EmitterInterface, EmitCallback, StealsDescription, short_name
 )
 from mypyc.ir.rtypes import RType,  bool_rprimitive
 
+# TODO: comment, we don't need error_kind since C functions are all ERR_MAGIC
+# however, other fields may need further investigation
+LLOpDescription = NamedTuple(
```

Comment:
This name doesn't fit the current implementation. Can you propose a new name that communicates that this specifically describes how to call a C function?

Reply:
I am using `CFunctionDescription` in the updated commits.